Traceback (most recent call last):
  File "I:\coding\ACK\arch-api\manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "I:\coding\ACK\arch-api\manage.py", line 22, in <module>
    main()
    ~~~~^^
  File "I:\coding\ACK\arch-api\manage.py", line 13, in main
    raise ImportError(
    ...<3 lines>...
    ) from exc
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Traceback (most recent call last):
  File "I:\coding\ACK\arch-api\manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "I:\coding\ACK\arch-api\manage.py", line 22, in <module>
    main()
    ~~~~^^
  File "I:\coding\ACK\arch-api\manage.py", line 13, in main
    raise ImportError(
    ...<3 lines>...
    ) from exc
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Traceback (most recent call last):
  File "I:\coding\ACK\arch-api\manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "I:\coding\ACK\arch-api\manage.py", line 22, in <module>
    main()
    ~~~~^^
  File "I:\coding\ACK\arch-api\manage.py", line 13, in main
    raise ImportError(
    ...<3 lines>...
    ) from exc
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2909
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2909
Not Found: /storage/files/Setting/favicon-1749030783.png
"GET /storage/files/Setting/favicon-1749030783.png HTTP/1.1" 404 3008
Not Found: /
"GET / HTTP/1.1" 404 2858
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [ArchdeaconryListView]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [CongregationListView]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [CongregationsByArchdeaconryView]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [DashboardAnalytics]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [ParishListView]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [RecordsListView]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
I:\coding\ACK\arch-api\core_apps\analyzer\views.py: Error [WorkbookUploadView]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2909
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /
"GET / HTTP/1.1" 404 2858
Not Found: /
"GET / HTTP/1.1" 404 2858
